html/pfappserver/root/static.alt/src/components/pfFormPrefixMultiplier.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <b-form-group horizontal :label-cols="(columnLabel) ? labelCols : 0" :label="$t(columnLabel)" 
-    :state="isValid()" :invalid-feedback="getInvalidFeedback()" :class="['prefixmultiplier-element', { 'mb-0': !columnLabel }, { 'is-focus': focus}]">
+    :state="isValid()" :invalid-feedback="getInvalidFeedback()" 
+    class="prefixmultiplier-element" :class="[{ 'mb-0': !columnLabel }, { 'is-focus': focus}]">
```

Comment:
You can simplify to a single object literal: `:class="{ 'mb-0': !columnLabel, 'is-focus': focus }"`